README.md

```diff
@@ -42,16 +42,19 @@ mkdir -p /var/tmp/reva/root/{home,oc}
 
 ## Quickstart
 
-After building the binary with the above steps and also setting up the prerequisites, run this command to start all services:
+- Make sure that the binary was built with the above steps.
+- Now start all services with the following command
 
 ```console
 ./bin/ocis server
 ```
 
-Open [https://localhost:9200](https://localhost:9200) and login using one of the demo accounts:
+- Open [https://localhost:9200](https://localhost:9200)
+- Accept the self-signed certificate (it is regenerated every time the server starts)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/gh/owncloud/ocis/pullRequest?prid=5426122)

Reply:
look, and now it's asking me to add 2 spaces again